docker/Dockerfile

```diff
@@ -1,12 +1,15 @@
 ARG BASE_IMAGE=ubuntu:22.04
 FROM "$BASE_IMAGE" AS base
+ARG TARGETARCH
```

Comment:
Why do we even care about target arch?
uname is enough because arm apt won't run on x86 cpu